pyaedt/modules/Boundary.py

```diff
@@ -3350,3 +3351,738 @@ def delete(self):
+        >>> network = pyaedt.modules.Boundary.Network(app)
+        >>> network.add_internal_node("TestNode", {"Type": "Transient",
+        >>>                                        "Function": "Linear", "Values": ["0.01W", "1"]})
+        """
+        if self._app.solution_type != "SteadyState" and mass is None and specific_heat is None:
+            self._app.logger.warning("The solution is transient but neither mass nor specific heat is assigned.")
+        if self._app.solution_type == "SteadyState" and (mass is not None or specific_heat is not None):
+            self._app.logger.warning(
+                "The solution is steady state so neither mass nor specific heat assignment is relevant."
+            )
+        if isinstance(power, (int, float)):
+            power = str(power) + "W"
+        props_dict = {"Power": power}
+        if mass is not None:
+            if isinstance(mass, (int, float)):
+                mass = str(mass) + "kg"
+            props_dict.update({"Mass": mass})
+        if specific_heat is not None:
+            if isinstance(specific_heat, (int, float)):
+                specific_heat = str(specific_heat) + "J_per_Kelkg"
+            props_dict.update({"SpecificHeat": specific_heat})
+        new_node = self._Node(name, self._app, node_type="InternalNode", props=props_dict, network=self)
+        self._nodes.append(new_node)
+        self._add_to_props(new_node)
+        return new_node
+
+    @pyaedt_function_handler()
+    def add_boundary_node(self, name, assignment_type, value):
+        """
+
+        Parameters
+        ----------
+        name: str
+            String containing the name of the node.
+        assignment_type: str
+            String containing the type assignment. It can be ``"Power"`` or ``"Temperature"``.
+        value: str or float or dict
+            String or float or dict containing the value of the assignment. If a float is passed ``"W"``
+             or ``"cel"`` is used according to ``assignment_type``. If ``type`` is ``"Power"`, a
+             dictionary can be passed to use temperature dependent or transient assignment.
+
+        Returns
+        -------
+        bool
+            True if successful.
+
+        Examples
+        --------
+
+        >>> import pyaedt
+        >>> app = pyaedt.Icepak()
+        >>> network = pyaedt.modules.Boundary.Network(app)
+        >>> network.add_boundary_node("TestNode", "Temperature", 2)
+        >>> ds = app.create_dataset1d_design("Test_DataSet", [1, 2, 3], [3, 4, 5])
+        >>> network.add_boundary_node("TestNode", "Power", {"Type": "Temp Dep",
+        >>>                                                       "Function": "Piecewise Linear",
+        >>>                                                       "Values": "Test_DataSet"})
+        """
+        if assignment_type not in ["Power", "Temperature"]:
+            raise AttributeError('``type`` can be only ``"Power"`` or ``"Temperature"``')
```

Comment:
Error messages should be complete sentences, concluding with ending punctuation.